fixed-point/Compiler/setting-up.txt

```diff
@@ -0,0 +1,21 @@
+
+1.	Download antlr4 runtime from the official website:
+	antlr-4.7-complete.jar
+	This is used to compile the grammar and generate the necessary visitor python classes.
+
+	java -jar antlr-4.7-complete.jar -Dlanguage=Python3 -visitor -no-listener flt.g4
+
+2.	Install the antlr4 python runtime from pip:
+	
+	Installing pip for python3:
+	sudo apt-get install python3-pip
+	sudo pip3 install --upgrade pip
+
+	Installing antlr runtime:
+	sudo pip3 install antlr4-python3-runtime
+
+3.	Install numpy from pip:
+	sudo pip3 install numpy
+
+4.	Run flt:
+	python3 Main.py ex-1.flt
```

Comment:
Main has to main (small m)

Reply:
This file shouldn't be there. It was a setup file which I had.
I will just remove it